docs/ksql-clickstream-demo/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# https://confluentinc.atlassian.net/browse/KSQL-292
+
+ARG DOCKER_REGISTRY
+
+FROM ${DOCKER_REGISTRY}confluentinc/docker-demo-base:3.3.0
```

Comment:
The version string `3.3.0` is outdated I think?  cc @bluemonk3y

Reply:
I would have thought it was 4.0.0

Reply:
changed to `demo-base:3.3.1`